website/docs/Conditions.md

```diff
@@ -0,0 +1,231 @@
+---
+slug: conditions
+---
+
+# Conditions
+Several types of access conditions can be defined:
+
+- EVM - on-chain state, eg NFT ownership, ETH balance, tx status, contract function call
+- RPC - ethereum RPC calls as defined in the [Official API](https://ethereum.org/en/developers/docs/apis/json-rpc/#json-rpc-methods)
+- Timelock - time-based conditions, eg Block Height
+
+We provide many helper objects to streamline the creation of common conditions.
+An expressive API also allows much more granular control of conditions, and we will provide examples of both methods wherever possible.
+
+## `Conditions.ERC721Ownership`
+`Conditions.ERC721Ownership` is a shortcut for building conditions that test for ownership of a specific ERC721 token (NFT):
+
+```js
+const NFTOwnership = new Conditions.ERC721Ownership({
+    contractAddress: '0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D',
+    parameters: [5954],
+});
+```
+
+If we want to be more verbose we can use `Conditions.Condition`.
+The above and below examples are completely equivalent:
+
+```js
+const NFTOwnershipConfig = {
+    contractAddress: "0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D",
+    standardContractType: "ERC721",
+    chain:  "ethereum",
+    method: "ownerOf",
+    parameters: [
+      "5954"
+    ],
+    returnValueTest: {
+      comparator: "==",
+      value: ":userAddress"
+    }
+  }
+const NFTOwnership = new Conditions.Condition(NFTOwnershipConfig)
+```
+## `Conditions.ERC721Balance`
+`Conditions.ERC721Balance` is a shortcut for building conditions that test for ownership of at least one ERC721 token (NFT) within a collection.
+
+```js
+const NFTBalance = new Conditions.ERC721Balance({
+    contractAddress: '0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D',
+});
+```
+Alternatively:
+```js
+const NFTBalanceConfig = {
+    contractAddress: "0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D",
+    standardContractType: "ERC721",
+    chain:  "ethereum",
+    method: "balanceOf",
+    parameters: [
+      ":userAddress"
+    ],
+    returnValueTest: {
+      comparator: ">",
+      value: "0"
+    }
+  }
+
+const NFTBalance = new Conditions.Condition(NFTBalanceConfig)
+```
+
+## `Conditions.TimelockCondition`
+`Conditions.TimelockCondition` is a shortcut for building conditions that test against block height.
+
+```js
+const timelock = Conditions.TimelockCondition({
+  returnValueTest: {
+    comparator: '>',
+    value: '100'
+  }
+});
+```
+
+or:
+
+```js
+const timelockConfig = {
+    contractAddress: "",
+    standardContractType: "",
+    chain:  "ethereum",
```

Comment:
I don't think Timelock currently takes a chain - cc @KPrasch . Unless this more along the lines of what it should be.